source/endpoints/parameters.rst

```diff
@@ -80,7 +80,81 @@ Examples:
    *  ``/users?sort=-username`` return users alphabetically from last to first by username (reverse order)
    *  ``/roles?sort=-id`` roles from last to first id
 
+.. _get-filters:
+
 Filters
 -------
 
-[TBD]
+Filters are one of the most powerful and versatile tools in BEdita API in order to retrieve only resources or objects you really need in your application.
+
+Filter expressions
+^^^^^^^^^^^^^^^^^^
+
+Generally every filter in a query string will have following syntax:
+
+  1. ``filter[{item}]={value}`` the simplest form, a filter item should be equal to some value
+  2. ``filter[{item}][{op}]={value}`` is a logical operation on filter value, where ``{op}`` can have following values:
+
+    * ``neq``, ``ne``, ``'!=`` or ``<>`` negation operator, meaning that filter item shoud not be equal to ``{{value}}``
+    * ``lt`` or ``<`` less than operator, meaning that filter item shoud be less than ``{{value}}``
+    * ``lte``, ``le`` or ``<=`` less than or equal operator; filter item shoud be less than or equal to ``{{value}}``
+    * ``gt`` or ``>`` greater than operator; filter item shoud be greater than ``{{value}}``
+    * ``gte``, ``ge`` or ``>=`` greater than or equal operator; filter item should be greater than or equal to ``{{value}}``
+
+  3. ``filter[{item}][]={value}`` array sintax: add a value to an array representing possible values of a filter item
+
+Filter expressions can be combined using ``&`` separator as many times as you want, limited only by the URL size.
+
+Too many ``filter`` combinations may of course result in unwantend or meaningless results, use them with caution :)
+
+Field filter
+^^^^^^^^^^^^
+
+The most common and simplest filter: retrieve only resources that have a field equal to some value, or greater/less than some value.
```

Comment:
I'm not quite sure of this, but I think that the correct form would be: "The most common and simple …".

Since "simple" is composed of two syllabes "simplest" is correct, but since there is "most" already in the sentence sounds quite redundant to me. 🤔  If we use a synonym of "simple", the sentence becomes "the most common and most manageable", and in this case I would omit the second "most", that's why I think that the second superlative is redundant.

Again: not really sure about this… 🤔